Dockerfile.tests

```diff
@@ -36,5 +36,11 @@ ARG CACHEBUST=1
 #Running the tests
 RUN npm test
 
+#Add the infra test script
+ADD run_infra_tests.sh ./run_infra_tests.sh
+
+#Running the infra tests
+RUN ./run_infra_tests.sh
```

Comment:
This needs you to ensure that the script is executable set 755 on run_infra_tests.sh
chmod 755 run_infra_tests.sh

Reply:
I  have already made the file executable by running the specified command

Reply:
Ok just wanted to ensure it :+1: